pkg/objectstore/objectstore.go

```diff
@@ -90,6 +90,8 @@ type Directory interface {
 
 	// Serialize directory
 	String() string
+
+	GetRegion() string
```

Comment:
Can this be added to the `objectstore.Bucket` interface and let the `bucket` struct implements it?

Reply:
I tried this initially but because of the weird relationship between bucket & directory it made me implement in directory also; then I realised I can remove the bucket one and have it only in directory

Reply:
or I should say, iirc, because we return Directory interface rather than Bucket from some funcs...will need to look again Tomorrow to find the funcs causing the issues